core/new-gui/src/app/workspace/component/property-editor/property-editor.component.html

```diff
@@ -12,5 +14,29 @@ <h3 class="texera-workspace-property-editor-title" mat-subheader>
       [layout]="formLayout"
       (onChanges)="onFormChanges($event)">
     </json-schema-form>
+
+
+    <ng-template #htmlContent>
+      <div class="container" *ngFor="let key of property_description | keyvalue">
+        <div class="property-detail-title">
+          <b>{{key.key}}</b>
+        </div>
+        <!-- <hr> -->
+        <div class="property-detail-description">
+          {{key.value}}
+        </div>
+        <!-- <hr> -->
+      </div>
```

Comment:
Rename the `key` of `property_description` to something like `operator_attribute`.

Remove the commented `<hr>` tags